src/PercentileStats.cpp

```diff
@@ -201,12 +202,19 @@ QRealList PercentileStats::percentileSums()
     {
         int percentile = qMax( 1, (int) ceil( i / percentileSize ) );
 
-        sums[ percentile ] += _data.at(i);
+        sums._ind[ percentile ] += _data.at(i);
     }
 
+    qreal runningTotal = 0;
```

Comment:
Please add an empty line between the `for (...)` and the preceding statement.
And please add a space between the `for` and the opening parenthesis: `for ( ... )`, not `for( ... )`.